core/patina_stacktrace/resolve_stacktrace/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "resolve_stacktrace"
+version = "0.1.0"
+edition = "2024"
+
+[dependencies]
+pdb = "0.8.0"
+pdb-addr2line = "0.11.2"
+comfy-table = "7.1.4"
```

Comment:
Since these are not coming from the workspace `Cargo.toml`, can you add a brief comment explaining that's intentional?

I think it's fine to not place these at the workspace-level, but the comment would clarify that these specific dependencies are explicitly separate instead of following the normal pattern in case others are wondering what to do with other dependencies in the future.